postgresql-centos7-atomicapp/Nulecule

```diff
@@ -6,6 +6,10 @@ metadata:
   name: PostgreSQL Atomic App
   appversion: 1.0.0
   description: This is PostgreSQL
+params:
+    - name: provider
+      description: The specified default provider.
+      default: docker
```

Comment:
This was because I thought there is some error with the postgres example using kubernetes so I reported to you about this and then you removed answers.conf that was causing the problem, that's why used docker there then https://github.com/projectatomic/nulecule-library/pull/70 if you want it to be changed I can do so.